.devcontainer/Dockerfile

```diff
@@ -44,7 +44,7 @@ RUN mkdir -p /etc/sudoers.d && echo 'Defaults env_keep += "SSH_AUTH_SOCK"' > /et
 
 # Add vscode user to clab_admins group so that it can run sudo-less clab commands
 # the group is created when clab is installed via the installation script
-RUN usermod -aG clab_admins vscode
```

Comment:
the group creation happens when containerlab is installed via the installer script on ln 22 of this dockerfile